packages/nest/src/oauth/application/oauth2.base.model.ts

```diff
@@ -133,38 +144,36 @@ export class OAuth2BaseModel implements ClientCredentialsModel {
                 ),
                 user,
             );
+
             if (callback) {
                 callback(null, final);
             }
+
             return final;
         } catch (err) {
             if (callback) {
                 callback(err, undefined);
             }
+
             return null;
         }
     }
 
-    async verifyScope(
-        token: Token,
-        scope: string | string[],
-        callback?: (err?: any, result?: boolean) => void,
-    ): Promise<boolean> {
+    public async verifyScope(token: Token, scope: string | string[], callback?: Callback<boolean>): Promise<boolean> {
         // console.log('WARNING: Scope Verification not implemented! Always returning true!');
```

Comment:
Clean-up

Reply:
Removed in 03be4951877ce3fac6daa66674fb5d9e9ac6bbe9